geojson_modelica_translator/system_parameters/schema.json

```
@@ -429,92 +459,264 @@
+          "description": "Nominal cooling water inlet temperature. (K)",
+          "type": "number",
+          "default": 308.15
+        },
+        "cooling_tower_water_temperature_difference_nominal": {
+          "Nominal water temperature difference between of the tower. (K)",
+          "type": "number",
+          "default": 280.15
+        },
+        "delta_temp_approach": {
+          "Approach tempearture difference. (K)",
+          "type": "number",
+          "default": 3
+        },
+        "ratio_water_air_nominal": {
+          "water_air_ratio_nominal",
+          "type": "number",
+          "default": 0.625
+        },
+
       },
       "required": [
-        "Q_flow_nominal",
-        "mCHW_flow_nominal",
-        "mCW_flow_nominal",
-        "dpCHW_nominal",
-        "dpCW_nominal",
-        "dpSetPoi",
-        "TCHWSet",
-        "dpCHWPum_nominal",
-        "dpCWPum_nominal"
+        "heat_flow_nominal",
+        "mass_chw_flow_nominal",
+        "chiller_water_flow_minimum",
+        "mass_cw_flow_nominal",
+        "pressure_drop_chw_nominal",
+        "pressure_drop_cw_nominal",
+        "pressure_drop_setpoint",
+        "temp_setpoint_chw",
+        "pressure_drop_chw_valve_nominal",
+        "pressure_drop_cw_pum_nominal",
+        "temp_air_wb_nominal",
+        "temp_cw_in_nominal",
+        "cooling_tower_water_temperature_difference_nominal",
+        "delta_temp_approach",
+        "ratio_water_air_nominal"
+      ]
+    },
+    "central_heating_plant_parameters": {
+      "description": "Central heating plant with maximum number of two boilers. Parameters associated with the model",
+      "type": "object",
+      "properties": {
+
+        "heat_flow_nominal": {
+          "description": "Nominal district heating load. (Watts)",
+          "type": "number",
+          "default": 8000
+        },
+        "mass_hw_flow_nominal": {
+          "description": "Nominal heating water mass flow rate. (kg/s)",
+          "type": "number",
+          "default": 10
+        },
+        "boiler_water_flow_minimum": {
+          "description": " Boiler minimum  water mass flow rate. (kg/s)",
+          "type": "number",
+          "default": 10
+        },
+        "pressure_drop_hw_nominal": {
+          "description": "Nominal heating water (boiler side) pressure drop. (Pa)",
+          "type": "number",
+          "default": 55000
+        },
+        "pressure_drop_setpoint": {
+          "description": "The heating water circuit pressure drop setpoint. (Pa)",
+          "type": "number",
+          "default": 50000
+        },
+        "temp_setpoint_hw": {
+          "description": "District circuit heating water temperature setpoint. (K)",
+          "type": "number",
+          "default": 338.15
+        },
+        "pressure_drop_hw_valve_nominal": {
+          "description": "Chiller isolation valve pressure drop. (Pa)",
+          "type": "number",
+          "default": 6000
+        },
```

```suggestion
        }
```
JSON doesn't allow trailing commas